### ISSUE TITLE
The 'Message' document refers to a document that doesn't exist 

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1696]
I just checked and GNB version wasn't in my list of download documents. Has it been removed?

Also it mentioned the KVJA. Why is it included there?

Last, the PEV translation is very Message like. I think it might be more like the Message than the GNB. Perhaps change the Message entry so it refers to the PEV.

![IMG_20210912_150659.jpg](https://user-images.githubusercontent.com/13920678/132973008-57c78274-d7e1-4310-8a8f-bf81d5b0804d.jpg)

**Comment by tuomas2** [comment]
KJVA is the versification system that is displayed in parenthesis for each document. Probably we shouldn't show versification system like that, it's confusing.

**Comment by cmahte** [comment]
The "Translation for Translators" is complete and similar to the message,
but far closer to scripture (it is a translation and not paraphrase,
although it makes no attempt to be word-word, but instead focuses solely on
providing the meaning that a reader from the time of the author would have
understood.) It is available in Ebible.org repository, and is complete.

The "Unfolding Simplified Text" is a work based on the T4T, with the
inline notes about metaphors, etc. removed, then edited for readability.
It also is similar to the goal of The message, but is the work of a
committee of sorts (which means it's style shifted a bit to read more like
the NIV, ESV, etc.) However, it is still undergoing final revisions.  It is
not on Sword repositories but is available at
https://git.door43.org/unfoldingWord/en_ust .

The "Dynamic English Bible" is a fork I made from the UST when it was still
the UDB, when it was still slightly more like the Message, and slightly
less like the NIV in the way it reads.  It is also not on Sword, but is
available at https://git.door43.org/mikey/en_deb .  The version there is
publishable, but at some point I intended to provide the Section Titles
that T4T has, edited to be shorter titles and not descriptions like the
T4T.
